### DISCUSSION TITLE
Question: orphanAgentUser

### DISCUSSION THREAD

**Discussion by mtran-88** [Ylianst/MeshCentral#6185] [Q&A] [answered]
Hi, I've tried to add orphanAgentUser entry under the domain section, but when I ran meshcentral, it says "ERROR: Unable to parse C:\mc\meshcentral-data\config.json.  Does this option still exist?  Thx!

**Comment by si458** [comment, marked as answer]
Yes it does.
Personally always use http://jsonlint.com to validate your json if it says it can't phase your json file

**Reply by mtran-88** [reply]
Thanks!
